NEWS.md

```diff
@@ -1,2 +1,7 @@
-## 1.0.0 Unreleased
-   * Project setup ([EDGDEMATIC-5](https://issues.folio.org/browse/EDGDEMATIC-5))
+## 1.0.0 - Released
```

Comment:
Please add `## 1.1.0 - Unreleased` in the top of document

Reply:
Added